frontend/services/entities/queries.ts

```diff
@@ -118,14 +118,7 @@ export default {
         query_id: queryId,
         selected,
       });
-      return Promise.resolve({
-        ...campaign,
-        hosts_count: {
```

Comment:
These empty values, and new ones added in this PR, already exist in the default state that the results of this API call are used to set, making this modification of the returned promise redundant.